tests/unicode.util.ts

```diff
@@ -0,0 +1,159 @@
+//these tests are weird, possibly incorrect or just tailored differently. we skip them.
+
+export let skipTests = [
```

Comment:
not sure how they are weird?

Reply:
From original line breaker, not sure how they're weird either...